### PR TITLE
feat: React NavBar and Login UI #40950

### PR DESCRIPTION
**Pull request by Priyankatiyar**
Created a new feature branch 40950.
[refer Task](https://redmine.successive.tech/issues/40950)
submit
errLogin
LoginEmail
navbar

### CODE REVIEW

File: training/src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    width: 350,
+    marginTop: theme.spacing(10),
+    marginLeft: theme.spacing(62),
+  },
+  icon: {
+    background: 'red',
+    marginLeft: theme.spacing(19),
+    marginTop: theme.spacing(2),
+  },
+
```
Comment (shashank-nx) [inline training/src/pages/Login/Login.jsx]: Remove Extra line.

File: training/src/pages/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+});
+
+class Login extends React.Component {
+  schema = yup.object().shape({
+    email: yup.string().trim().email().required('Email is a required field'),
+    password: yup.string().required('Password is a required field').matches(/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.{8,})/, 'Must contain 8 characters, at least one uppercase letter, one lowercase letter and one number'),
+
```
Comment (shashank-nx) [inline training/src/pages/Login/Login.jsx]: Remove Extra line.

**Review by shashank-nx — CHANGES_REQUESTED** [review]

File: training/src/pages/Login/Login.jsx (hunk 2 above)
Comment (Priyankatiyar) [reply training/src/pages/Login/Login.jsx]: Done.

**Review by shashank-nx — APPROVED** [review]
Looks fine to me

**Pull request merged**